v4/weaviate/backup/backup_creator.go

```diff
@@ -0,0 +1,51 @@
+package backup
+
+import (
+	"context"
+
+	"github.com/semi-technologies/weaviate/entities/models"
+)
+
+type BackupCreator struct {
+	helper            *backupCreateHelper
+	includeClasses    []string
+	excludeClasses    []string
+	storageName       string
+	backupID          string
+	waitForCompletion bool
+}
+
+func (c *BackupCreator) WithIncludeClasses(classes ...string) *BackupCreator {
```

Comment:
Java and JS clients have method named `WithIncludeClassNames(classNames)`.
Usually term className is used instead of class in go (and other) clients. So methods are named WithClassName(className), no WithClass(class)